client/src/main/java/gui/playing/PlayBoardController.java

```diff
@@ -59,7 +61,7 @@
   private Sea sea;
   private LanguageVersion languageVersion = new LanguageVersion();
   private OutputChannelDispatcher outputChannelDispatcher = new OutputChannelDispatcher();
-
+  private Map<String,String> messageMap = new HashMap<>();
```

Comment:
what about Enum, String map?

Reply:
I don't think it's necessary to create additional variables to use them only to retrieve values from a map